src/chapters/02/main.tex

```diff
@@ -125,6 +134,7 @@ \section{Theory}\label{sec:theory}
 \(P_{t}\) where \((P_{t})_{ij}\) is the probability of being in state \(j\)
 after \(t\) time units having started in state \(i\). We can use \(Q\) to
 calculate \(P_{t}\) using the matrix exponential:
+%%% A matrix exponential is not a standard concept. Should this be explained? Or at least talked about like it doesn't matter if you don't know. (again: who is this book for?)
```

Comment:
> is not a standard concept.

What is the standard concept is the correct question (matrix exponential is absolutely standard).

When we figure these things out we need to keep in mind further chapters where some levels of detail might significantly lengthen/change the book

Reply:
Once we agree we should capture these decisions as principles that we can add to `src/README.md` on #66 which in turn can be written in an "about this book" section.

Reply:
Agreed action: Add footnote.